src/locale/locales/de/messages.po

```diff
@@ -133,39 +128,39 @@ msgstr "Zu meinen Feeds hinzufügen"
 #: src/view/com/modals/ListAddRemoveUsers.tsx:191
 #: src/view/com/modals/UserAddRemoveLists.tsx:128
 msgid "Added to list"
-msgstr "Den Listen hinzufügen"
+msgstr "Zur Liste hinzugefügt"
 
 #: src/view/screens/PreferencesHomeFeed.tsx:164
 msgid "Adjust the number of likes a reply must have to be shown in your feed."
-msgstr "Passen Sie die Anzahl der Likes an, die eine Antwort haben muss, um in Ihrem Feed angezeigt zu werden."
+msgstr "Passe die Anzahl der Likes an, die eine Antwort haben muss, um in deinem Feed angezeigt zu werden."
 
 #: src/view/com/modals/SelfLabel.tsx:75
 msgid "Adult Content"
 msgstr "Inhalt für Erwachsene"
```

Comment:
Sounds kinda weird to me. Maybe "Inhalt nicht für Kinder und Jugendliche geeignet"? Depends on the context, I guess.